### PR TITLE
 Hide Download button in the navbar when at the top of the page

### PR DESCRIPTION
**Pull request by Zuhdannur**
Hello 🙌
I wanted to help this project

Issue: Hide Download button in the navbar when at the top of the page

### CODE REVIEW

**Comment by shahednasser** [comment]
Hi @Zuhdannur I like your approach, but can you do it with css instead? basically it should be like `.navbar:not(.scrolling) .downloadIcon` display should be none, and for `.navbar.scrolling .downloadIcon` display should inline-block

**Comment by Zuhdannur** [comment]
hi @shahednasser thanks for the reviews, you can check it newest commit

File: assets/js/index.js [hunk 1]
```diff
@@ -52,8 +52,8 @@ $(document).ready(function () {
       $(".navbar").addClass("scrolling");
       $(".sidebar").addClass("scrolling");
     } else {
-      $(".navbar").removeClass("scrolling");
-      $(".sidebar").removeClass("scrolling");
+      $(".downloadIcon").css('display','inline-block')
```
Comment (shahednasser) [inline assets/js/index.js:55]: Change in this file is unnecessary now. Please change it back to how it was

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by Zuhdannur** [comment]
please check it again

**Review by shahednasser — CHANGES_REQUESTED** [review]
Can you please remove added files in ".idea" folder?

**Pull request merged**